inspirehep/modules/literaturesuggest/static/js/literaturesuggest/data_sources/data_source.js

```diff
@@ -104,6 +104,7 @@ define(function(require, exports, module) {
           url: this.url + id
         })
         .always(function(data) {
+          depositionType = data.query.type;
```

Comment:
maybe this is not yet complete, at the moment you are making the variable global and then you are not using it?